ui/desktop/app/routes/scopes/scope/projects/sessions/session.js

```diff
@@ -12,6 +12,11 @@ export default class ScopesScopeProjectsSessionsSessionRoute extends Route {
   @service store;
 
   // =methods
+  resetController(controller, isExiting, transition) {
```

Comment:
None blocking comment: would you mind adding a comment to describe what resetController does?